packages/@aws-cdk/aws-certificatemanager/test/test.dns-validated-certificate.ts

```diff
@@ -112,4 +112,31 @@ export = {
     test.throws(() => expect(stack), /DNS zone hello.com is not authoritative for certificate domain name example.com/);
     test.done();
   },
+
```

Comment:
This test doesn't seem to verify the new behavior (we need to verify that there's an error thrown when you synthesize)

Reply:
Sorry but i don't think i follow you, i built the test like the previous ones on this test suite, but i am totally inexperienced with writing TS and TS tests so might have missed something?

Reply:
You are right, my bad. Looks good.